### ISSUE TITLE
Does not work with Python 3

### ISSUE THREAD

**Issue by EmilStenstrom** [ionelmc/pypi-alias#4]
```bash
$ python --version
Python 3.9.0

$ pypi-alias
Traceback (most recent call last):
  File "/home/emilstenstrom/.pyenv/versions/django-components/bin/pypi-alias", line 8, in <module>
    sys.exit(main())
  File "/home/emilstenstrom/.pyenv/versions/3.9.0/envs/django-components/lib/python3.9/site-packages/pypi_alias/__main__.py", line 15, in main
    assert 'python' not in name
TypeError: a bytes-like object is required, not 'str'
```

**Comment by EmilStenstrom** [comment]
I'll fix it too in a PR if this project is still active?

**Comment by ionelmc** [comment]
Now it's active :-) Please make a PR if you want.

**Comment by EmilStenstrom** [comment]
@ionelmc Excellent :) Do you need backwards compatibility with Python 2?

**Comment by ionelmc** [comment]
Not really, there's absolutely no reason to support Python 2 in cli tooling nowadays. Tho I wounld't spend time fixing up the setup.py cause I'll just reapply cookiecutter-pylibrary before a release anyway (and it has that stuff sorted out).

**Comment by ionelmc** [comment]
Released 1.0 just now.
